crates/pairwise_nostd_internal/src/reducer.rs

```diff
@@ -241,3 +242,56 @@ impl Reducer for Mean {
         value[[Mean::VALUE_WEIGHT]] = accum_state[Mean::WEIGHT];
     }
 }
+
+// TODO: refactor so that Histogram doesn't hold a vector and move to
+//       pairwise_internal/accumulator.rs (to )
```

Comment:
Is this comment left over from the copy-paste

Reply:
Thanks